docs/dense-retrieval.md

```diff
@@ -0,0 +1,35 @@
+# Direct PyPI Package Replication on Dense Retrieval
+
+It's easy to replicate runs on many "standard" IR test collections directly from the PyPI package (i.e., with only `pip install`)!
+
+## MS MARCO Passage Ranking
+
+MS MARCO passage ranking task, dense retrieval with TCT-ColBERT, HNSW index.
+```bash
+$ python -m pyserini.dsearch --topics msmarco_passage_dev_subset --index msmarco-passage-tct_colbert-hnsw --query_emb msmarco-passage-dev-subset-tct_colbert --output run.msmarco-passage.tct_colbert.hnsw.txt
+```
+
+To evaluate:
+
+```bash
+$ wget -O jtreceval-0.0.5-jar-with-dependencies.jar https://search.maven.org/remotecontent?filepath=uk/ac/gla/dcs/terrierteam/jtreceval/0.0.5/jtreceval-0.0.5-jar-with-dependencies.jar
```

Comment:
Since we assume the user has a dev checkout, we can use the scripts in `tools/` - `tools/eval/trec_eval.9.0.4/trec_eval`